src/api/response/iiif/presentation-api/metadata.js

```diff
@@ -45,10 +45,6 @@ function metadataLabelFields(source) {
       label: "Genre",
       value: source.genre.map((item) => item.label),
     },
-    {
-      label: "Keyword",
-      value: source.keywords,
-    },
```

Comment:
Do we want to remove this here?   Are regular API (non IIIF) functions using this helper too I wonder?